lib/rest_connection/rightscale/server_interface.rb

```diff
@@ -229,38 +198,32 @@ def stop
   end
 
   def launch
-    return @impl.launch if @multicloud
-    return @impl.start unless @multicloud
+    return @impl.launch
   end
 
   def terminate
-    return @impl.terminate if @multicloud
-    return @impl.stop unless @multicloud
+    return @impl.terminate
   end
 
   def start_ebs
-    return connection.logger("WARNING: Gateway Servers do not support start_ebs. Ignoring.") if @multicloud
-    return @impl.start_ebs unless @multicloud
+    return connection.logger("WARNING: Gateway Servers do not support start_ebs. Ignoring.")
```

Comment:
Hmm, this should probably be fixed to actually work instead of just printing out an obsolete warning. The stop/start tests will no longer work otherwise. Same for all of the following method as well.